website/docs/api/curated-transformer.mdx

```diff
@@ -0,0 +1,546 @@
+---
```

Comment:
We typically avoid hyphens in the name of the page, so I'd suggest renaming to `/api/curatedtransformer.mdx`.

Also, we need to add this page to the index of the API pages or it won't show up in the menu on the left.

Reply:
Renamed and added.